### PR TITLE
Fix default labels in issue template

### PR DESCRIPTION
**Pull request by akihironitta**
## What does this PR do?
Addresses https://github.com/PyTorchLightning/pytorch-lightning/pull/12403#issuecomment-1076989464.

Thank you @carmocca for pointing it out :]

### Does your PR introduce any breaking changes? If yes, please list them.
None

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [n/a] Did you make sure to **update the documentation** with your changes? (if necessary)
- [n/a] Did you write any **new necessary tests**? (not for typos and docs)
- [n/a] Did you verify new and **existing tests pass** locally with your changes?
- [n/a] Did you list all the **breaking changes** introduced by this pull request?
- [x] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃 My bad 😅

cc @carmocca @akihironitta @borda

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name: Bug report
 about: Create a bug report to help us improve
 title: ''
-label: needs triage
+labels: needs triage
```
Comment (akihironitta) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: Reviewers: No need to put `"` here as we didn't have it in the past https://github.com/PyTorchLightning/pytorch-lightning/blob/0e6ee9c39dfb1608eb3ffef67f962a725ad935a5/.github/ISSUE_TEMPLATE/bug_report.md?plain=1.

**Review by carmocca — APPROVED** [review]
3rd time is the charm!

**Review by krshrimali — APPROVED** [review]
🚀

**Comment by akihironitta** [comment]
oops, could I have your approval again...? @carmocca

**Review by Borda — APPROVED** [review]

**Pull request merged**